### DISCUSSION TITLE
Auto position component toolbar

### DISCUSSION THREAD

**Discussion by josejbracho** [GrapesJS/grapesjs#5553] [Q&A]
In the website demo, there's a behaviour I would like to emulate.

The component toolbar auto positions on top or bottom depending on the scroll, like so: 

At the top when there's enough room:

<img width="1762" alt="Screenshot 2023-12-12 at 3 45 35 PM" src="https://github.com/GrapesJS/grapesjs/assets/83281328/1cb702bb-ae44-4820-b65b-a99f7f1f9f4b">

At the bottom after scrolling and not enough room:

<img width="1760" alt="Screenshot 2023-12-12 at 3 45 42 PM" src="https://github.com/GrapesJS/grapesjs/assets/83281328/6c447779-634c-470c-a79e-fc870bdeb4ae">

What controls this behaviour, how can I apply it to my editor?

Thanks in advance.

**Comment by ronaldohoch** [comment]
i'm running a plugin here using the last version of grapesjs and it has the same behaviour, maybe your grapesjs are in a different version.
